_specifications/lsp/3.18/types/command.md

```diff
@@ -8,10 +8,17 @@ interface Command {
 	 * Title of the command, like `save`.
 	 */
 	title: string;
+
+	/**
+	 * An optional tooltip.
+	 */
+	tooltip?: string;
```

Comment:
Does spec want to use a "tooltip" wording that seems to be UI-centric? Would it be better to say, for example, "description" and let client choose to use it in the tooltip or elsewhere?